configs/env-example

```diff
@@ -20,5 +20,21 @@ KEYCLOAK_REALM=
 KEYCLOAK_CLIENT=
 
 # E2E Test
-E2E_TEST_STUDENT_PASSWORD=ADD_THE_PASSWORD_HERE_TO_RUN_LOCALLY
 E2E_TEST_STUDENT_USERNAME=student_e2e_test
+E2E_TEST_STUDENT_PASSWORD=ADD_THE_PASSWORD_HERE_TO_RUN_LOCALLY
+
+E2E_TEST_BCeID_USERNAME=SIMS_E2ETests
+E2E_TEST_BCeID_PASSWORD=ADD_THE_PASSWORD_HERE_TO_RUN_LOCALLY
```

Comment:
As we hav called line 24 as STUDENT_PASSWORD, i think we shold call this  INSTITUTION_PASSWORD. Same for Username

Reply:
This user/password is not related to an institution. This user could be used to any other E2E test that requires a valid BCeID User.
Under this scenario, do you still believe that we should change it? If yes, I would avoid adding "institution" to it.